storage.ml

```diff
@@ -9,24 +9,26 @@ let current_version = 7
    5 cookie has two new fields last_access and user_agent
    6 tokens has 3 new fields: name, last_access and usage_count
    7 added unikernel_updates to keep track of when unikernels are updated
+   8 we now store a list of configurations
 *)
 
-let t_to_json users configuration =
+let t_to_json users configurations =
   `Assoc
     [
       ("version", `Int current_version);
       ("users", `List (List.map User_model.user_to_json users));
-      ("configuration", Configuration.to_json configuration);
+      ("configurations", `List (List.map Configuration.to_json configurations));
     ]
 
 let t_of_json json =
   match json with
   | `Assoc xs -> (
       let ( let* ) = Result.bind in
-      match (get "version" xs, get "users" xs, get "configuration" xs) with
-      | Some (`Int v), Some (`List users), Some configuration ->
+      match (get "version" xs, get "users" xs, get "configurations" xs) with
+      | Some (`Int v), Some (`List users), Some (`List configurations) ->
```

Comment:
For older versions (<8) we need to preserve the case where `get "configuration"` is used, and it results in a `configuration` (i.e. not a list).